oil_lang/expr_eval.py

```diff
@@ -33,8 +33,9 @@
     scope_e, scope_t,
     part_value, part_value_t,
     lvalue,
-    value, value_e, value_t, value__Bool, value__Int, value__Float,
-    value__Str, value__MaybeStrArray, value__AssocArray, value__Obj
+    value, value_e, value_t, value__AssocArray, value__Bool, value__Dict,
+    value__Int, value__Float, value__List, value__MaybeStrArray, value__Str,
```

Comment:
just FYI, I made a change to allow removing all these silly imports

Instead you just import `value, value_e, value_t`, and that's it


https://oilshell.zulipchat.com/#narrow/stream/208950-zephyr-asdl/topic/Remove.20value__Foo.20types.2C.20etc.2E

We have to migrate gradually though, separately from this change